README.md

```diff
@@ -398,3 +399,22 @@ I'm kwf most places on the internet, you may know me as Kenny Foner offline.
 
    I'm interested on collaborating on pretty much anything and I'm happy to pair
    with people of any experience.
+
+## vaibhavsagar
+
+1. Availability
+
+   I have a fairly standard 9am-6pm job, and I'd be happy to pair from 8pm my
+   time onwards or early in the morning if that suits you better.
+
+2. Experience
+
+   I've been using Haskell seriously for about two years but not
+   professionally.  I'm a maintainer of the IHaskell project and have
+   contributed numerous typo fixes to Haskell projects such as Cabal and GHC.
```

Comment:
> I'm a maintainer of the IHaskell project

Hey, that's great! I really like the idea of IHaskell with Jupyter for a supercharged GHCi REPL, but I very often had problems setting it up, running without 'kernel' problems, or loading 3rd party libraries.

Would you happen to know if something like the following is possible with nix-shell+IHaskell+Jupyter+libs?

`nix-shell -p "haskell.packages.ghc822.ghcWithPackages (p: with p; [hint compose-ltr pretty-simple lens dump time-qq overload generics-eot])" --run 'ghci'`

Reply:
Have a look at the [`release.nix`](https://github.com/gibiansky/IHaskell/blob/d53ecedb4ea8f047535410cc61985f9c5bfa2ec0/release.nix) in the IHaskell repository, you should be able to use it to set up an IHaskell environment. I have third-party libraries set up like [this](https://github.com/vaibhavsagar/notebooks/blob/master/git-from-scratch/default.nix). Happy to demonstrate in a pairing session or something like that :smile:

Reply:
Hmm, a demonstration would be quite cool! And after the demonstration, if you are willing, we could try to set it up over on my side too. And if we are lucky all this could take no more than 10-20 minutes. I'm looking forward, let's have a call sometime soon :)